src/funcs/getDbSchemaStructures.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const getSchemas = require('../queries/getSchemas');
+const getTablesInSchema = require('../queries/getTablesInSchema');
+const getTableStructure = require('../queries/getTableStructure');
+const getConstraints = require('../queries/getConstraints');
+
+const getPrimaryKey = (schema, tableName, constraints) => {
+  if (!constraints) return undefined;
+  return constraints.find(({ constraintType, fromSchema, fromTable }) => {
+    return fromSchema === schema &&
+      fromTable === tableName &&
+      constraintType === "PRIMARY KEY";
+  });
+}
+
+/**
+ * @function getDbSchemaStructures
+ * @returns {array[object]} of all schemas of shape
+ * [
+ *   {  
+ *     schema: 'public',
+ *     tables: [
+ *       {
+ *          tableName: 'users'
+ *          structure: [ ... ] // of pg column definitions
+ *       }
+ *     ]
+ *   }
+ * ]
+ */
```

Comment:
Do we need commented out code?

Reply:
this is a note, since the package is not in typescript, to describe the shape of the data returned. I think it should stay.

Reply:
do believe this is a comment to show what the expected return would be.